docs/claim.md

```diff
@@ -59,11 +59,11 @@ Depending on the option you chose in the first step you will be presented with o
 
 2. Sign a message through external wallet.
 
-    If you have typed in your ethereum address you will need to sign a message through external wallet that you used to buy the tokens and own a private key to. Once you have signed your message, paste the signed message into the response field.
+    If you have typed in your ethereum address you will need to sign a message through external wallet that you used to buy the tokens and own a private key to. Once you have signed your message, paste the signature (starting with "0x") of the signed message into the response field.
```

Comment:
You can also paste whole json there. But I guess it's ok? It should work both ways

Reply:
Yes, I didn't want to overcomplicate it, let's see if ppl will be asking we can change that